docs/reference/linstorsatelliteconfiguration.md

```diff
@@ -78,6 +78,10 @@ the matching key. Available types are:
 * `lvmThin`: Configures a [LVM Thin Pool](https://man7.org/linux/man-pages/man7/lvmthin.7.html) as storage pool.
   Defaults to using the storage pool name as name for the thin pool volume and the storage pool name prefixed by
   `linstor_` as the VG name. Can be overridden by setting `thinPool` and `volumeGroup`.
+* `file`: Configures a file system based storage pool. Configures a host directory as location for the volume files.
```

Comment:
We need to make it extra clear that `file:` contains the path to a directory, if set. That is a bit counter-intuitive. One would expect something named "file" to be a filename or a path to a file.

We could even change the key to `filePool` or something to clarify this.

Perhaps include an example with a path too?

Reply:
It would currently look like this:

```
file:
  directory: /var/lib/linstor-pools/foo
```

But I agree that is confusing. I somehow don't like `filePool`, because then you have the repeated pool `storagePools: [ filePool: ... ]`, but that is a minor issue. Would `fileSystem` be an alternative?

Reply:
> It would currently look like this:

That is at least a bit clearer than I had thought.

`fileSystem` could work. `fileBacked`? Is there any precedent in the linstor docs? I can't see any with a quick search.

Reply:
> Is there any precedent in the linstor docs

Doesn't look like it:

> Create a file storage pool

and

> File / FileThin: Mostly used for demonstration / experiments. LINSTOR will basically reserve a file in a given directory and will configure a [loop device](https://man7.org/linux/man-pages/man4/loop.4.html) on top of that file.

I don't think there is anything to take from that.

---

`fileBacked` sounds good.

Would it then be `fileBackedThin`, `thinFileBacked`, `sparseFileBacked`, `fileThinBacked`? I don't really like any of these. Perhaps it's better to have it as an option on the `fileBacked` struct? I just mirrored the way it is configured for LVM, because the options there are actually different.

Reply:
My preference is actually `filePool` and `fileThinPool`. That avoids introducing a new term in this context. Shame about the aesthetics :man_shrugging:

Reply:
> Maybe we should do that

Done